src/dune_rules/lib_flags.ml

```diff
@@ -85,10 +85,10 @@ let link_deps sctx t mode =
 module L = struct
   type nonrec t = Lib.t list
 
-  let to_iflags dirs =
+  let to_iflags ?(flag = "-I") dirs =
```

Comment:
In our codebase we often try to avoid default flags, as the defaults are often easy to miss and in this case "to_iflags" which generates `-H` flags is somewhat strange.

As such I suggest making this a labelled `to_flag` function and 
```ocaml
let to_ifags = to_flags "-I"
let to_hflags = to_flags "-H"
```

This prevents having to change all the code while allowing an easy way to get an `-H` flag.

Reply:
You might not even need to expose `to_hflags` in the `mli` if nobody outside the module is going to use it. That way the compiler can warn about unused functions in case the code stops using `to_hflags`.

Reply:
You are right for now `to_hflags` is not used anywhere outside `lib_flags.mli`, but I exposed it for the sake of symmetry (since if `to_iflags` is to be used somewhere in some later work, `to_hflags` would be also needed).

But maybe it is better for now to remove it from the interface.